From 3349c5a75225649114173529de180bd6d46a9c6e Mon Sep 17 00:00:00 2001
From: Lukas Slebodnik <lslebodn@redhat.com>
Date: Fri, 16 Jan 2015 16:45:21 +0100
Subject: BUILD: Fix building with automake 1.15

The macro AC_BUILD_AUX_DIR was used too late. As a result of this automake 1.15
generated configure script which ignored the directory build/ and was not able
to find missing files.

    configure: error: cannot find install-sh, install.sh,
    or shtool in "." "./.." "./../..".

After removing macro AC_BUILD_AUX_DIR, autoreconf will install auxiliary files
into $srcdir.

Signed-off-by: Lukas Slebodnik <lslebodn@redhat.com>
Reviewed-by: Simo Sorce <simo@redhat.com>

diff --git a/proxy/Makefile.am b/proxy/Makefile.am
index b5ba2af..86b5933 100644
--- a/proxy/Makefile.am
+++ b/proxy/Makefile.am
@@ -79,8 +79,6 @@ AM_CPPFLAGS = \
     -DSYSCONFDIR=\"$(sysconfdir)\" \
     -DLOCALEDIR=\"$(localedir)\"
 
-EXTRA_DIST = build/config.rpath
-
 GSS_PROXY_LIBS = $(POPT_LIBS) $(KRB5_LIBS) $(VERTO_LIBS) $(INI_LIBS) $(GSSAPI_LIBS) $(GSSRPC_LIBS)
 
 if BUILD_SELINUX
diff --git a/proxy/configure.ac b/proxy/configure.ac
index a709d20..5e43c3b 100644
--- a/proxy/configure.ac
+++ b/proxy/configure.ac
@@ -12,7 +12,6 @@ m4_ifdef([AC_USE_SYSTEM_EXTENSIONS],
 CFLAGS="$CFLAGS -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE"
 
 AC_CONFIG_SRCDIR([BUILD.txt])
-AC_CONFIG_AUX_DIR([build])
 
 AM_INIT_AUTOMAKE([-Wall foreign subdir-objects tar-pax])
 AM_PROG_CC_C_O
-- 
cgit v0.10.2


